### PR TITLE
Explanation of Scrum (agile methodology)

### PR DESCRIPTION
**Pull request by seant1**
#13
This is way over my 300(?) word limit but will cut it down in the final edit. You can also use/take out any content from my section if it's relevant to yours (probably not but hey).

### CODE REVIEW

File: Report.md [hunk 1]
```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
 
 
 ## **Agile Methodology - Scrum**
-4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
+> 4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
```
Comment (zachartmann) [inline Report.md]: Are we keeping these in the final product?

File: Report.md [hunk 2]
```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
...
+#### Product Owner – Zac
+The product owner is an individual responsible for establishing and communicating the product vision including defining and prioritising features to be developed. The product owner must make quick and informed decisions while monitoring project progress and release dates.
+#### Scrum Master – Aiswarya
+The scrum master facilitates scrum team events and promotes scrum theory, rules, and practices.
+#### Development Team - Anastasia, Sanya, Sean
+The development team is responsible for building the deliverable increment of a product in each sprint.
+
+### Events/Activities
+
+A sprint is a fixed period of time after which a deliveralble iteration of a product must be developed. The following activities were conducted in each of our team's 1-week sprints (Sunday to Friday).
```
Comment (zachartmann) [inline Report.md]:
```suggestion
A sprint is a fixed period of time after which a deliverable iteration of a product must be developed. The following activities were conducted in each of our team's 1-week sprints (Sunday to Friday).
```

File: Report.md [hunk 3]
```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
...
+
+#### Product Owner – Zac
+The product owner is an individual responsible for establishing and communicating the product vision including defining and prioritising features to be developed. The product owner must make quick and informed decisions while monitoring project progress and release dates.
+#### Scrum Master – Aiswarya
+The scrum master facilitates scrum team events and promotes scrum theory, rules, and practices.
+#### Development Team - Anastasia, Sanya, Sean
+The development team is responsible for building the deliverable increment of a product in each sprint.
+
+### Events/Activities
+
```
Comment (zachartmann) [inline Report.md]:
```suggestion
```

File: Report.md [hunk 4]
```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
...
+Our team conducted sprint planning sessions on the first day of each sprint (Sunday). In these meetings we established the goals for the current sprint, and reviewed the stories/issues in the backlog. The tasks for each sprint were allocated to members of the development team and moved from the “Backlog” column (on project board) to the “Ready for Dev” column.
+#### Sprint Review
+Sprint reviews were held every Friday for members of the development team to explain and deliver their work for the given sprint. The product owner and other team members evaluated what was completed and determined the progress made and tasks left to complete.
+#### Sprint Retrospective
+Our sprint retrospectives were held after each sprint review to re-evaluate our workflow, communication, and any other issues that arose during the sprint.
+#### Daily Scrum
+Due to the clashing schedules of our team members, we decided to communicate/conduct daily scrum meetings over text. Each day, progress updates and issues were discussed and any blockers were collectively reviewed and resolved.
+
+### Artefacts
+
```
Comment (zachartmann) [inline Report.md]:
```suggestion
```

File: Report.md [hunk 5]
```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
...
+Artefact | Description
+---|---
+Product Backlog | An ordered list of all tasks/features/stories/requirements for a product (managed by product owner)
+Sprint Backlog | A subset of the Product Backlog for development in a sprint (managed by development team)
+Product Increment | A potentially shippable version of the product that builds on the previous increment/version
+Sprint Burndown Chart | A graphical representation of the development team's progress over time (backlog items completed)
+
+explain "done" and "timebox"?
```
Comment (zachartmann) [inline Report.md]: Probably not necessary. Done doesn't seem like it's too important. You could add a definition of timebox up when you're defining scrum in general - would add words if we need it.

**Review by zachartmann — CHANGES_REQUESTED** [review]
Great job Sean, just a few like empty lines and a typo.
Don't think we need to talk about done and timebox if you don't want to (re: comment)

File: Report.md (hunk 2 above)
Comment (zachartmann) [reply Report.md]: There's an empty line here. Looking at the actual markdown I can't tell the difference so on second thought it's probably no big deal at all - just OCD more than anything

File: Report.md (hunk 5 above)
Comment (seant1) [reply Report.md]: I'm already way over my word limit so.. probably not... but maybe

File: Report.md (hunk 1 above)
Comment (seant1) [reply Report.md]: I think we'll remove it

**Review by zachartmann — APPROVED** [review]
Looks good to merge btw

**Pull request merged**